experimental/dds/tree2/src/test/feature-libraries/defaultFieldKinds.spec.ts

```diff
@@ -174,22 +206,27 @@ describe("Value field changesets", () => {
 			change1WithChildChange,
 		);
 
-		const expected: FieldKindsTypes.ValueChangeset = {
-			value: { set: tree1 },
-			changes: nodeChange1,
+		const expected: FieldKindsTypes.OptionalChangeset = {
+			fieldChange: {
+				newContent: { set: tree1, changes: nodeChange1 },
+				id: brand(1),
+				wasEmpty: false,
+				revision: change1.revision,
+			},
 		};
 
 		assert.deepEqual(change1WithChildChange, expected);
-		assert.deepEqual(
-			fieldHandler.rebaser.compose(
-				[makeAnonChange(childChange1), change1],
-				simpleChildComposer,
-				idAllocator,
-				crossFieldManager,
-				defaultRevisionMetadataFromChanges([change1]),
-			),
-			change1.change,
+		const composition = fieldHandler.rebaser.compose(
+			[makeAnonChange(childChange1), change1],
+			simpleChildComposer,
+			idAllocator,
+			crossFieldManager,
+			defaultRevisionMetadataFromChanges([change1]),
 		);
+		assert.deepEqual(composition, {
+			fieldChange: { ...change1.change.fieldChange, revision: change1.revision },
+			childChange: nodeChange1,
+		});
```

Comment:
I'm not sure if I translated this test correctly. This assertion gave me a bit of trouble and I essentially just used the output of the composition to make this test pass.

Reply:
I think that is correct. That's a case where the improvements you made in the past to the sequence field now accrue to the value field.

Reply:
I thought that was the case, but wanted to be sure! Thanks!